src/app/modules/analytics-config/components/groups/groups.component.ts

```diff
@@ -56,14 +56,13 @@ export class GroupsComponent implements OnInit, OnDestroy {
   public constructor(
     private defaultValuesService: DefaultValuesService,
     public dialog: MatDialog,
-    private arlasStartupService: ArlasStartupService,
-    private configService: ArlasConfigService,
     private analyticsInitService: AnalyticsInitService,
     private translate: TranslateService,
     private cdr: ChangeDetectorRef,
     private colorService: ArlasColorService,
```

Comment:
ColorService is declared twice

Reply:
I'll treat the refactor of color service in a seperated PR